File: GestionRendezVous/src/Entity/Consultation.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity(repositoryClass="App\Repository\ConsultationRepository")
 */
class Consultation
{
    /**
     * @ORM\Id()
     * @ORM\GeneratedValue()
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="date")
     */
    private $Date;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $Heure;

    /**
     * @ORM\ManyToOne(targetEntity="App\Entity\Medecin")
     * @ORM\JoinColumn(nullable=false)
     */
    private $idMedecin;

    /**
     * @ORM\Column(type="boolean", nullable=true)
     */
    private $estValidee;

    /**
     * @ORM\Column(type="string", length=255, nullable=true)
     */
    private $nom;

    /**
     * @ORM\Column(type="string", length=255, nullable=true)
     */
    private $prenom;

    /**
     * @ORM\ManyToOne(targetEntity="App\Entity\Patient")
     */
    private $Patient;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->Date;
    }

    public function setDate(\DateTimeInterface $Date): self
    {
        $this->Date = $Date;

        return $this;
    }

    public function getHeure(): ?string
    {
        return $this->Heure;
    }

    public function setHeure(?string $Heure): self
    {
        $this->Heure = $Heure;

        return $this;
    }

    public function getIdMedecin(): ?Medecin
    {
        return $this->idMedecin;
    }

    public function setIdMedecin(?Medecin $idMedecin): self
    {
        $this->idMedecin = $idMedecin;

        return $this;
    }

    public function getEstValidee(): ?bool
    {
        return $this->estValidee;
    }

    public function setEstValidee(?bool $estValidee): self
    {
        $this->estValidee = $estValidee;

        return $this;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(?string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getPrenom(): ?string
    {
        return $this->prenom;
    }

    public function setPrenom(?string $prenom): self
    {
        $this->prenom = $prenom;

        return $this;
    }

    public function getPatient(): ?Patient
    {
        return $this->Patient;
    }

    public function setPatient(?Patient $Patient): self
    {
        $this->Patient = $Patient;

        return $this;
    }
}

File: Android/GestionRDV/app/src/main/java/com/example/sio2018/gestionrdv/Consultation.java
package com.example.sio2018.gestionrdv;

import java.time.LocalDate;
import java.util.Date;

public class Consultation {
    private int id;
    private int idMedecin;
    private String date;
    private int heure;
    private String nom;
    private String prenom;


    public Consultation(int pid, int pidMedecin,  String pdate, int pheure, String pnom, String pprenom){
        this.id = pid;
        this.idMedecin = pidMedecin;
        this.date = pdate;
        this.heure = pheure;
        this.nom = pnom;
        this.prenom = pprenom;
    }

    public int getId(){
        return id;
    }

    public int getIdMedecin(){
        return idMedecin;
    }

    public String getDate(){
        return date;
    }

    public int getHeure(){
        return heure;
    }

    public String getNom(){
        return nom;
    }

    public String getPrenom(){
        return prenom;
    }
}
